plugins/plugin-maven/che-plugin-maven-server/src/main/java/org/eclipse/che/plugin/maven/server/execution/ProcessHandler.java

```diff
@@ -160,6 +161,7 @@ private void notifyOnTerminating() {
     }
 
     private void notifyOnText(String text, ProcessOutputType type) {
+        LOG.debug(type.name() + " " + text);
```

Comment:
string concatenation shouldn't be used in DEBUG traces (give object as parameters and use kind of MessageFormat